cmd/tailscale/cli/debug.go

```diff
@@ -451,6 +457,64 @@ func runLocalCreds(ctx context.Context, args []string) error {
 	return nil
 }
 
+func looksLikeHTTPMethod(s string) bool {
+	if len(s) > len("OPTIONS") {
+		return false
+	}
+	for _, r := range s {
+		if r < 'A' || r > 'Z' {
+			return false
+		}
+	}
+	return true
+}
+
+func runRawLocalAPI(ctx context.Context, args []string) error {
+	if len(args) == 0 {
+		return errors.New("expected at least one argument")
+	}
+	method := "GET"
+	if looksLikeHTTPMethod(args[0]) {
+		method = args[0]
+		args = args[1:]
+		if len(args) == 0 {
+			return errors.New("expected at least one argument after method")
+		}
+	}
+	path := args[0]
+	if !strings.HasPrefix(path, "/localapi/") {
+		if !strings.Contains(path, "/") {
+			path = "/localapi/v0/" + path
+		} else {
+			path = "/localapi/" + path
+		}
+	}
+
+	var body io.Reader
+	if len(args) > 1 {
```

Comment:
Default/optionally read from stdin?

```suggestion
	if len(args) < 2 || args[1] == "-" {
		body = os.Stdin // or maybe read all and store in a bytes.Buffer so NewRequest can sniff Content-Length?
	} else {
```

Reply:
I tried to make it automatic but there are too many heuristic. Otherwise it sometimes blocks waiting for a body when none is arriving. So I went with "-" only to specify one.